src/averageAges.js

```diff
@@ -14,8 +14,24 @@
  *
  * @return {number}
  */
+
+/* Global function */
+const calcAverage = function(peopleData) {
+  const BornYear = peopleData
+    .reduce((bornSum, { born }) => bornSum + born, 0);
+  const DiedYear = peopleData
+    .reduce((diedSum, { died }) => diedSum + died, 0);
```

Comment:
it could be done easier. You don't need to calculate the sum of the born and death years. You could calculate the sum of the ages of people right away using one reduce method instead of two